_projects/sparcv8leon3-gr716-mimas/board_config.h

```diff
@@ -16,59 +16,58 @@
 
 #define SYSCLK_FREQ (100 * 1000 * 1000) /* MHz */
 
-#define UART_MAX_CNT 6
+#define UART_MAX_CNT 2
 
 #define UART_BAUDRATE 115200
 
-#define UART0_BASE ((void *)0x80300000)
-#define UART1_BASE ((void *)0x80301000)
-#define UART2_BASE ((void *)0x80302000)
-#define UART3_BASE ((void *)0x80303000)
-#define UART4_BASE ((void *)0x80304000)
-#define UART5_BASE ((void *)0x80305000)
+#define UART0_BASE ((void *)0x80302000)
+#define UART1_BASE ((void *)0x80303000)
+#define UART2_BASE ((void *)-1)
+#define UART3_BASE ((void *)-1)
+#define UART4_BASE ((void *)-1)
+#define UART5_BASE ((void *)-1)
```

Comment:
How about `NULL` instead? `(void *)-1` seem kinda hackish to me

Reply:
I'll do (void *)0, I don't think we want to include anything in `board_config.h``